### PR TITLE
Fix Prow recommended policy test

### PR DESCRIPTION
**Pull request by RedbackThomson**
Description of changes:
Fixes build and test failures in the Prow cluster for the recommended policy test:
- Adds a new Make target for the test entrypoint
- Switches the tests to use the standard integration test container
- Fixes the Prow config namespace in the Flux `Kustomization` to allow resources outside the `prow` namespace

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: prow/jobs/jinja/presubmits/service_tests.jinja2 [hunk 1]
```diff
@@ -78,13 +80,13 @@
     spec:
       serviceAccountName: pre-submit-service-account
       containers:
-      - image: public.ecr.aws/bitnami/aws-cli:2.2.43
+      - image: public.ecr.aws/m5q3e4b2/prow:prow-integration-0.0.7
```
Comment (RedbackThomson) [inline prow/jobs/jinja/presubmits/service_tests.jinja2:83]: Note to self: this needs to use the image context

**Review by A-Hilaly — APPROVED** [review]

File: prow/jobs/jinja/presubmits/service_tests.jinja2 (hunk 1 above)
Comment (jaypipes) [reply prow/jobs/jinja/presubmits/service_tests.jinja2:83]: what is the image context?

**Review by jaypipes — APPROVED** [review]
:+1:

**Pull request merged**